apps/src/music/player/sequencer/Simple2Sequencer.ts

```diff
@@ -310,6 +317,12 @@ export default class Simple2Sequencer extends Sequencer {
   }
 
   private addNewEvent<T extends PlaybackEvent>(event: T) {
+    if (this.currentEventCount >= MaxNumberEvents) {
+      console.log(`Exceeded ${MaxNumberEvents} events`);
```

Comment:
It looks like this can cause a flood of console logs:

<img width="359" alt="image" src="https://github.com/user-attachments/assets/76c115f8-d481-46a9-bf53-722cce9911b8">
<img width="545" alt="image" src="https://github.com/user-attachments/assets/0e9bf3b6-a48d-41fd-bccf-b98ebe69825c">

Are there more efficient way we could be logging this?

I was going to propose a follow-up of using a workspace alert banner to let the user know what's going on. Could that be useful here?

Reply:
To add - might be nice to store a flag in the sequencer that we can flip as soon as we cross the limit? That way we can surface it to the UI as well and update components accordingly, i.e. showing a workspace banner, disabling the play button/timeline, etc. And we can use that flag to show the console warning just the first time the limit is crossed.